.ci_support/osx64.yaml

```diff
@@ -12,3 +12,5 @@ channel_sources:
   - conda-forge
 target_platform:
   - osx-64
+
+
```

Comment:
Can you please revert this change and the removed recipe? Please editing only the recipe you are adding.

Reply:
Hi Sorry for the confusion but I'm not sure what to do.
I ran the script `build-locally.py` which deleted 2 directories and modified this file to point to a local /opt path.
I will try starting over.

Reply:
I tried to revert all changes made by `build-locally.py` `.git diff main` now only lists my new recipe.

@conda-forge-admin, please ping https://github.com/orgs/conda-forge/teams/help-python